ui/src/timeMachine/components/Vis.tsx

```diff
@@ -132,7 +130,10 @@ const TimeMachineVis: SFC<Props> = ({
 const mstp = (state: AppState): StateProps => {
   const {
     isViewingRawData,
-    view: {properties: viewProperties},
+    view: {
+      properties: viewProperties,
+      properties: {queries},
```

Comment:
that's so weird looking!

Reply:
> this: `query.match(/v.\S*/g).map(m => m.slice(2))` what is this jibberish?

Reply:
oh i'm sorry. didn't realize you weren't fluent in genius.